content/300-accelerate/400-api-reference.mdx

```diff
@@ -113,3 +113,9 @@ interface AccelerateInfo {
 | `region`       | `String`                                                     | The data center region that received the request.                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
 | `requestId`    | `String`                                                     | Unique identifier of the request. Useful for troubleshooting.                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
 | `signature`    | `String`                                                     | The unique signature of the Prisma operation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+
+## Errors
+
+Accelerate-related errors start with `P6xxx`.
```

Comment:
```suggestion
Prisma Accelerate-related errors start with `P6xxx`.
```

Reply:
I don't like how this looks but ChatGPT says it's the way to go 😂 

<img width="571" alt="image" src="https://github.com/prisma/docs/assets/4058327/d74b0180-816f-4c6b-81a5-1ceb32aba030">